### ISSUE TITLE
Show doi in IPT metadata

### ISSUE THREAD

**Issue by kbraak** [gbif/ipt#1151]
```
What feature would like to see being added to the IPT?

Now that DOIs minted for the resource are supported. It will be useful for that IPT
to reflect those in its metadata. I suppose it would probably best be put in citation
rather than alternate identifier. 

Being able to see (and set) this in the IPT will take the citation information one
step closer to the source, so that any aggregator or other user could take advantage
of and propagate it.
```

Original issue reported on code.google.com by `gtuco.btuco` on 2015-03-26 14:39:00

**Comment by peterdesmet** [comment]
It looks that this is already available no?

* One can manually add DOI to `resource citation identifier`. If auto-generation is on for `resource citation`, it will be included there as well.
* The dataset DOI can be populated by the API in the DwC field `datasetID`

Or did you mean something else @tucotuco?

**Comment by kbraak** [comment]
When a dataset with no existing DOI assigned to it gets registered with GBIF, GBIF assigns it a DOI. That GBIF-DOI resolves to a GBIF Dataset Page, not the IPT Dataset Page. 

After registration with GBIF, the IPT should automatically update the dataset with its newly assigned GBIF-DOI. This will save the manager from having to fill it in manually.

Please note that as per the [IPT versioning policy](https://github.com/gbif/ipt/wiki/IPT2Versioning.wiki), publishers should assign a new DOI to the dataset each time it undergoes a scientifically significant change. This is only possible, however, if the IPT has been configured with a DataCite or EZID account. Otherwise, the dataset will retain the same GBIF-DOI forever hence the GBIF Portal does not handle dataset versioning.

**Comment by peterdesmet** [comment]
Any plans to implement this?

We now have our datasets harvested through our IPT DCAT feed and the identifiers that end up at the aggregator website are the GBIF UUID urls. We would rather have the DOIs there, but then they need to be available in the IPT. See https://github.com/inbo/ipt-dcat/issues/57

/cc @stijnvanhoey

**Comment by kbraak** [comment]
@peterdesmet there is no plan to implement this in the next release. 

I'd like to point out there is a danger in using the GBIF DOI as the data citation identifier hence it resolves to the version of the dataset that GBIF last indexed, not the original data in the IPT. 

In order for the data citation to enable other researchers to locate the original data (e.g. to reproduce results based on the original data) it would be more accurate to use the IPT URL as the citation identifier because it resolves to the original data. Nevertheless, a DOI is probably still preferred because it guarantees persistent access to the data over time given the IPT URL may change.

**Comment by peterdesmet** [comment]
Correct. Also GBIF page links to source data, although that link could be highlighted more. Plans to do this in new design?

**Comment by kbraak** [comment]
Indeed @MortenHofft 's new design of the dataset page raises the visibility of the original source data and will help users discover the origin of the dataset more easily.

**Comment by frafra** [comment]
IPT 2.5 has been released, but I do not see such functionality. Should the milestone be postponed?

**Comment by mike-podolskiy90** [comment]
Thank you for you question @frafra 
The 2.5 milestone was a general milestone with issues we initially planned to include to the 2.5.0 and others. I renamed it to 2.x a while ago to avoid confusion.
